packages/realm_dart/lib/src/handles/web/decimal128.dart

```diff
@@ -23,17 +43,94 @@ class Decimal128 implements intf.Decimal128 {
   static final negativeInfinity = -infinity;
 
   /// Parses a string into a [Decimal128]. Returns `null` if the string is not a valid [Decimal128].
-  static Decimal128? tryParse(String source) => webNotSupported();
+  static Decimal128? tryParse(String source) => Decimal.tryParse(source).convert(Decimal128._);
 
   /// Parses a string into a [Decimal128]. Throws a [FormatException] if the string is not a valid [Decimal128].
-  factory Decimal128.parse(String source) => webNotSupported();
+  factory Decimal128.parse(String source) => Decimal128._(Decimal.parse(source));
 
   /// Converts a `int` into a [Decimal128].
-  factory Decimal128.fromInt(int value) => webNotSupported();
+  factory Decimal128.fromInt(int value) => Decimal128._(Decimal.fromInt(value));
 
   /// Converts a `double` into a [Decimal128].
-  factory Decimal128.fromDouble(double value) => webNotSupported();
+  factory Decimal128.fromDouble(double value) {
```

Comment:
Can we take a shortcut here and do a `Decimal.parse(double.toString())`?

Reply:
Yes. Done.